### ISSUE TITLE
Can't read image from usb stick

### ISSUE THREAD

**Issue by OBch** [maxnet/berryboot#91]
I have formatted and installed multiple RPI OS from the internet on one SD card with BerryBoot.  Easier for me than NOOBS, so thank you very much for this product!  

Now I am trying to load Fedora_18_softfp.img192 from your sourceforge OS images on to  my usb stick.  Berryboot sees the usb as a potential target at sda1, but when I try to Add OS the only option is from the internet. On tty2, #fdisk /dev/sda1 shows Start=32 End=2015231 ID=7 System=hpfs/ntfs/exFAT.  When I hover my mouse over Add OS there is a prompt but nothing about loading an OS from the USB stick.  Thanks for answering.

**Comment by maxnet** [comment]
Hold down your left mouse button and keep it pressed over "add OS", don't just hoover.
USB stick needs to be FAT or EXT4. NTFS is not built-into the kernel (only as module)

**Comment by OBch** [comment]
Thank you for the very quick response, Max.  Last night I re-read the Walkthrough and used a different USB stick and was able to install the Fedora_18_softfp.img192 from your sourceforge OS images!  I was meaning to add an update last night, but was too tired.

There was a gotcha with that install - user/password.  On the tenth guess it proved to be user=root password=fedora.  Thanks for a wonderful product!

If possible on the next release of BerryBoot if the behavior of the left click would show both options (internet and usb) and/or change the "hover" prompt to show hint "Click and Hold Left mouse button for usb install", it would ease Newbee user frustration.  A slight change to the Walkthrough from [Hold down your mouse button over the “Add OS” button] to [Click and Hold down your Left mouse button over the “Add OS” button]

Thanks again for a fine product.  Mine was a classic variation of RTM.  However there is no proper manual.  I would like to volunteer to help write one, perhaps pdf format that could be available for download from sourceforge.  

In case I forgot to say it before.  Thanks!

**Comment by maxnet** [comment]
Good to hear you managed to solve it.
Might move the "from USB" option to the "select OS image to download from Internet" window some time in the future.

**Comment by OBch** [comment]
Hi Maxnet,

Thanks for your support and your great product.

Watched a youtube demo of your product by Patrick Norton - dyi-trying.  So decided to give it a try - good decision.

If you don't mind I'd like to see if I can put together a BootBerry manual for your perusal.  My laptop is Vista OS but I have VirtualBox installed so I took screenshots of a Debian install so I could remember what I did.  Windows screenshots to MS Word with some typed comments then printed out to a pdf_printer.  See attached...

  ----- Original Message -----
